saleor/graphql/account/utils.py

```diff
@@ -463,7 +463,7 @@ def look_for_permission_in_users_with_manage_staff(
 
 
 def requestor_has_access(
-    requestor: Union["User", "App"], owner: Optional["User"], perm
+    requestor: Union["User", "App"], owner: Optional["User"], *perms
```

Comment:
Please also update the docstring `perm` -> `perms`. We should also add some description that this function expects **one of** permissions to be assigned to return true.